db/src/test/java/DBServiceTest.java

```diff
@@ -48,6 +49,17 @@ public void insert_withHireDate_returnsHireDateAsToday() throws SQLException, Cl
         Assert.assertTrue(savedEmployee.getFirstName().equals(toSave.getFirstName()));
 
         //assertJ
-        assertThat(savedEmployee).isEqualToIgnoringGivenFields(toSave, "number");
+        assertThat(savedEmployee).isEqualToIgnoringGivenFields(toSave, "number", "hireDate");
+    }
+
+    @Test
+    public void listReturnsEmployeeClassObject() throws SQLException, ClassNotFoundException {
+        DBService dbService = new DBService();
+        List<Employee> employeeSalaryByDept = dbService.listEmployeeSalariesByDepartment();
+
```

Comment:
You should persist controlled data for each individual test and the run asserts over this data